plugin/pkg/shoot/quotavalidator/admission.go

```diff
@@ -345,18 +351,28 @@ func (q *QuotaValidator) determineRequiredResources(allocatedResources corev1.Re
 	return requiredResources, nil
 }
 
-func (q *QuotaValidator) getShootResources(shoot core.Shoot) (corev1.ResourceList, error) {
+func (q *QuotaValidator) getShootResources(shoot gardencorev1beta1.Shoot) (corev1.ResourceList, error) {
 	cloudProfile, err := q.cloudProfileLister.Get(shoot.Spec.CloudProfileName)
 	if err != nil {
 		return nil, apierrors.NewInternalError(fmt.Errorf("could not find referenced cloud profile: %+v", err.Error()))
 	}
 
+	coreCloudProfile := core.CloudProfile{}
+	if err := gardencorev1beta1.Convert_v1beta1_CloudProfile_To_core_CloudProfile(cloudProfile, &coreCloudProfile, nil); err != nil {
+		return nil, apierrors.NewInternalError(err)
```

Comment:
```suggestion
		return nil, apierrors.NewInternalError(fmt.Errorf("could not convert v1beta1 cloudProfile: %+v", err.Error()))
```
Similarly at other places, if any